### ISSUE TITLE
Run the base image and vim does not display properly

### ISSUE THREAD

**Issue by singlaive** [JAremko/alpine-vim#11]
It might be weird why I run the base image jare/alpine-vim rather than the vim-bundle image. I do so in order to learn how to create my own vim docker image from scratch and attempted to use the base image as a starting point.

I only need add one line before ENTRYPOINT statment of alpine-vim-base/Dockerfile:
WORKDIR /workspace

Build this image after I cd into the repo:
docker build -t docker-vim alpine-vim-base

And I run it for editing:
docker run -it --rm -v `pwd`:/workspace docker-vim alpine-vim-base/Dockerfile

Surprisingly, vim does not render all content. You can see 1/3 of lines are blank:
![image](https://user-images.githubusercontent.com/851206/49337789-d6e45d00-f610-11e8-830c-95c3d773bad8.png)

And if I run :set nu to display line number, the problem becomes more visible, and you can not even scroll up to go to the begining of the file:
![image](https://user-images.githubusercontent.com/851206/49337905-d220a880-f612-11e8-91d4-82db827108c1.png)

I must say this issue is intermittent, and one out of ten it might give me proper display. Please anyone can help me?
Environment: macOS Mojave 10.14.1
Docker: 
![image](https://user-images.githubusercontent.com/851206/49337937-a5b95c00-f613-11e8-8b1f-547082d32f8b.png)

**Comment by JAremko** [comment]
@singlaive  But if you build vim-bundle it works fine? I haven't re-built the images for a while. Mb it's a new issue with  next vim/deps versions.

**Comment by singlaive** [comment]
Yes vim-bundle is fine. I am not saying there is anything wrong with your Dockerfile or your repo, I think it is some pitfall about run vim in docker. The symptom is:
* Building vim-bundle ends up with solarised scheme not installed properly somehow. when run vim-bundle, it throw error about not finding solarised, then entering vim it displays fine. BUTTTTTT, if I explicitly update the Dockerfile for vim-bundle and install solarised theme thus no error, it does not display properly exactly like what presented above.
* Shrinking the Dockerfile of alpine-vim to be only those four lines and it still reproduces this issue:
From alpine
apk add vim
WORKDIR /workspace
ENTRYPOINT vim
* If you do docker run -it --entrypoint /bin/sh to get into the container first, then run vim inside, it displays fine.

It does look like it is a configureation issue only appearing when you run vim via docker, but I tried quite a few hours and cannot narrow down the root cause. Please help.

**Comment by JAremko** [comment]
@singlaive  Also try setting env variable  `TERM=xterm`  or mb `xterm-256color`  vial docker's  `-e` run-flag.
